src/components/views/messages/MessageActionBar.tsx

```diff
@@ -97,7 +97,10 @@ const OptionsButton: React.FC<IOptionsButtonProps> = ({
         <ContextMenuTooltipButton
             className="mx_MessageActionBar_maskButton mx_MessageActionBar_optionsButton"
             title={_t("Options")}
-            onClick={() => {
+            onClick={(e: React.MouseEvent) => {
```

Comment:
The style guide says we're supposed to avoid putting function literals inline in JSX, could this be moved to a variable?

Reply:
Sure!